CHANGELOG.latest.md

```diff
@@ -1,8 +1,13 @@
-### New Features
-* Add product type to typescript types (#507) via Cesar de la Vega (@vegaro)
-### Bugfixes
-* Add `product_plan_identifier` to `EntitlementInfo` model (#512) via Toni Rico (@tonidero)
+### Breaking Changes
+* Add in app messages API support (#510) via Toni Rico (@tonidero)
+* Temporarily remode DEFERRED proration mode in preparation for upgrade to Billing Client 6 (#506) via Mark Villacampa (@MarkVillacampa)
+* Update android to 7.0 (#520) via Toni Rico (@tonidero)
```

Comment:
How is this breaking? As a developer, what do I have to do? This doesn't really tell me anything. 

We should be very clear with anything that we call breaking. 

As far as I know, the actual things we need to explain are: 
- We've updated BillingClient to v6. If you are using observer mode, this means that you have to update your BillingClient version to v6 as well, or stick with an older version of the SDK. 
- DEFERRED proration mode support is temporarily disabled in this version. If you use DEFERRED proration mode, you should not upgrade to this version of the SDK yet, you should wait until support is re-added. 
- If the user has an active subscription and has billing issues, an in-app message will automatically be displayed. You can prevent this behavior by calling <insert name of> API.